pkg/networkplugin-syncer/handlers/ovn/router_ovn_cluster_north.go

```diff
@@ -45,6 +48,80 @@ func (ovn *SyncHandler) connectOvnClusterRouterToSubm() error {
 	return nil
 }
 
+// changeMgmtAllowRelatedACL modifies an ACL installed by ovn-kubernetes, which in it's default
+//                           form will prevent ICMP fragment packets getting back to pods, precluding
+//                           Path MTU discovery from working
+func (ovn *SyncHandler) changeMgmtAllowRelatedACL(chassisSwitch string) error {
+	mgmtPortIP, err := ovn.getMgmtPortIP(chassisSwitch)
+	if err != nil {
+		return err
+	}
+
+	aclList, err := ovn.nbdb.ACLList(chassisSwitch)
+	if err != nil {
+		return errors.Wrapf(err, "error getting ACL list for chassis switch: %q", chassisSwitch)
+	}
+
+	for _, acl := range aclList {
+		// If the ofending ACL is found, we switch it to a more neutral form that will allow our ICMP packet
+		if !(acl.Action == "allow-related" &&
+			acl.Match == "ip4.src=="+mgmtPortIP &&
+			acl.Direction == "to-lport" &&
+			acl.Priority == 1001) {
+			continue
+		}
+
+		klog.Infof("Fixing ovn-kubernetes host mgmt ACL in switch %q", chassisSwitch)
+
+		delCmd, err := ovn.nbdb.ACLDel(chassisSwitch, acl.Direction, acl.Match, acl.Priority, nil)
+		if err != nil {
+			return errors.Wrapf(err, "error building ACLDel cmd for %#v", acl)
+		}
+
+		err = ovn.nbdb.Execute(delCmd)
+		if err != nil {
+			return errors.Wrapf(err, "error executing ACLDel for allow-related in switch %q", chassisSwitch)
+		}
+
+		addCmd, err := ovn.nbdb.ACLAdd(chassisSwitch, acl.Direction, acl.Match, "allow",
+			acl.Priority, nil, false, "", "")
+		if err != nil {
+			return errors.Wrapf(err, "error creating ACLAdd cmd for %#v with allow", acl)
+		}
+
+		err = ovn.nbdb.Execute(addCmd)
+		if err != nil {
+			return errors.Wrapf(err, "error executing ACLDel & ACLAdd for allow-related in switch %q", chassisSwitch)
+		}
+
+		return nil
+	}
+
+	klog.V(log.DEBUG).Infof("ovn-kubernetes host mgmt ACL wasn't present as allow-related in switch %q", chassisSwitch)
+
+	return nil
+}
+
+func (ovn *SyncHandler) getMgmtPortIP(chassisSwitch string) (string, error) {
+	// NOTE: tried to grab k8s- constant from ovn-kubernetes but that pulls a lot of go mod dependencies
+	// which conflict with ours
+	port, err := ovn.nbdb.LSPGet("k8s-" + chassisSwitch)
```

Comment:
Make it a local constant and in a comment add link to file in ovn-kubernetes where it is originally defined. Useful for future reference and in case the constant changes in ovn-kubernetes.